packages/cozy-client/src/withMutation.jsx

```
@@ -1,39 +1,25 @@
-      this.client = props.client || context.client
-      if (!this.client) {
-        throw new Error(
-          `Could not find "client" in either the context or props of ${wrappedDisplayName}`
-        )
-      }
-    }
-
-    mutate = (...args) => {
-      return this.client.mutate(mutation.apply(null, args), options)
-    }
+    const mutate = (...args) => client.mutate(mutation.apply(null, args), options)
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)